front/app/components/UI/MultipleSelect/index.tsx

```diff
@@ -91,8 +91,10 @@ export default class MultipleSelect extends React.PureComponent<Props, State> {
           options={options}
           onChange={this.handleOnChange}
           isDisabled={disabled}
-          styles={selectStyles}
+          styles={selectStyles()}
+          menuPosition="fixed"
           menuPlacement="auto"
+          hideSelectedOptions
```

Comment:
Looks like a nice improvement: everything that's selected gets out of the list of options.